tsconfig.json

```diff
@@ -4,7 +4,7 @@
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
     "noImplicitAny": false,
-    "noUnusedLocals": false,
+    "noUnusedLocals": true,
```

Comment:
Is there an issue for the fundamental problem that causes this issue? It should be a lint error not a build error right?